templates/security/reset_password/request.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends '@Tabler/security.html.twig' %}
+
+{% block login_social_auth %}{% endblock %}
+
+{% block login_box_msg %}
+    {{ 'Forgot password'|trans({}, 'TablerBundle') }}
+{% endblock %}
+
+{% block login_form %}
+        {% for flashError in app.flashes('reset_password_error') %}
+            <div class="alert alert-danger" role="alert">{{ flashError }}</div>
+        {% endfor %}
+
+        <p class="text-muted mb-4">{{ 'Enter your email address and your password will be reset and emailed to you.'|trans({}, 'TablerBundle') }}</p>
+
+        {{ form_start(requestForm) }}
+
+        {{ form_row(requestForm.email) }}
+
+        <div class="form-footer">
+            <button type="submit" class="btn btn-primary w-100">
+                {{ tabler_icon('mail') }}
+                {{ 'Send me new password'|trans({}, 'TablerBundle') }}
```

Comment:
Please re-use existing translations `Reset your password` and `I forgot my password`